paper.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'A Python Project Template for Healthy Scientific Software'
+tags:
+  - Python
+  - template
+authors:
+  - name: Drew Oldag
+    orcid: 0000-0001-6984-8411
+    equal-contrib: true
+    affiliation: "1, 2"
+  - name: Melissa DeLucchi
+    orcid: 0000-0002-1074-2900
+    equal-contrib: true
+    affiliation: "1, 3"
+  - name: Wilson Beebe
+    orcid: 0009-0003-1791-8707
+    affiliation: "1, 2"
+  - name: Doug Branton
+    orcid: 0009-0009-7822-7110
+    affiliation: "1, 2"
+  - name: Sandro Campos
+    orcid: 0009-0007-9870-9032
+    affiliation: "1, 3"
+  - name: Carl Christofferson
+    orcid: 0009-0009-1164-3494
+    affiliation: "1, 2"
+  - name: Andrew Connolly
+    orcid: 0000-0001-5576-8189
+    affiliation: "1, 2"
+  - name: Jeremy Kubica
+    orcid: 0009-0009-2281-7031
+    affiliation: "1, 3"
+  - name: Olivia Lynn
+    orcid: 0000-0001-5028-146X
+    affiliation: "1, 3"
+  - name: Konstantin Malanchev
+    orcid: 0000-0001-7179-7406
+    affiliation: "1, 3"
+  - name: Alex Malz
+    orcid: 0000-0002-8676-1622
+    affiliation: "1, 3"
+  - name: Rachel Mandelbaum
+    orcid: 0000-0003-2271-1527
+    affiliation: "1, 3"
+  - name: Sean McGuire
+    orcid: 0009-0005-8764-2608
+    affiliation: "1, 3"
+  - name: Chris Wenneman
+    orcid: 0009-0002-6218-5007
+    affiliation: "1, 2"
+affiliations:
+  - name: LINCC-Frameworks
```

Comment:
Should not have a hyphen... just "LINCC Frameworks" (here and throughout)

Reply:
Thanks for pointing this out. I'll fix this up throughout the paper.

Reply:
Done